health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -54,6 +55,132 @@ export const UICustomizations = {
+                /> */}
+              </>
+            )
+            )
+            break;
+          case "CREATION_DATE":
+            let epoch = row?.auditDetails?.createdTime;
+            return Digit.DateUtils.ConvertEpochToDate(epoch);
+            // return row?.auditDetails?.createdTime;
+            break;
+          case "ACTION":
+            const tenantId = Digit.ULBService.getCurrentTenantId();
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)